README.md

```diff
@@ -49,16 +49,21 @@ Run `yarn gen:types` in root while `yarn dev` is running.
 
 This will deploy the application to AWS using CDK.
 
-##### Prerequisites:
+##### Cloud deployment prerequisites:
```

Comment:
i think someone mentioned it in the chat yesterday but probably worth adding a step for installing `yarn` itself